src/Microsoft.DotNet.Interactive/Logging/KernelEventLoggingFormatter.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Linq;
+using Microsoft.DotNet.Interactive.Events;
+using Microsoft.DotNet.Interactive.Formatting;
+
+namespace Microsoft.DotNet.Interactive
+{
+    internal sealed class KernelEventLoggingFormatter : LoggingFormatter<KernelEvent>
+    {
+        internal KernelEventLoggingFormatter() : base(FormatKernelEvent)
+        {
+        }
+
+        private static bool FormatKernelEvent(KernelEvent @event, FormatContext context)
+        {
+            if (@event is null)
+            {
+                context.Writer.Write($"{nameof(KernelEvent)}: <null>");
+            }
+            else
+            {
+                context.Writer.Write($"{@event.GetType().Name} (Command {@event.Command.ToDisplayString("text/log")}");
+
+                switch (@event)
```

Comment:
This should probably have a default case.

Reply:
For all events, we already log the event type name (and the associated command) on line 21 above. Is there any additional info we would want to log in the `default` case?

Reply:
Ah, I see.

Some sample log output including commands and events would probably be useful to add to the PR description.

Reply:
Good call - will do this soon.